Arguments needed to smoothly build modules without needing specific user config

diff --git a/jhbuild/defaults.jhbuildrc b/jhbuild/defaults.jhbuildrc
index f6e01308..cf583107 100644
--- a/jhbuild/defaults.jhbuildrc
+++ b/jhbuild/defaults.jhbuildrc
@@ -90,9 +90,12 @@ repos = {}
 cvsroots = {}
 svnroots = {}
 branches = {}
-module_autogenargs = {}
+module_autogenargs = {
+    'telepathy-mission-control': 'PYTHON=/usr/bin/python2',
+    'WebKit': 'PYTHON=/usr/bin/python2',
+}
 module_cmakeargs = {}
 module_mesonargs = {}
 module_makeargs = {}
 module_extra_env = {}
 module_makecheck = {}
